x-pack/plugins/lens/common/expressions/xy_chart/axis_config.ts

```diff
@@ -16,6 +16,12 @@ export interface AxesSettingsConfig {
   yRight: boolean;
 }
 
```

Comment:
I'm creating a comment here just to group the conversation under the same thread (is not a comment about that specific line of code):

Quoting the original issue: https://github.com/elastic/kibana/issues/113755
>When I have multiple axis
> And I have a reference line bound to each axis
> I need to match the axis to the threshold line color
> So I can easily see which threshold belongs to which axis
>2nd use case: also improve the integration on canvas where the background color is more likely to be something custom

The described case to me is really specific to a main use case: `multiple axis` and `reference line` or, less descriptive case: `within Canvas`

If we allow the user to freely choose the axis color (and/or the label color) we are achieving the following, unwanted (I hope) situation where:
1. color is a subjective preference, doesn't necessarily bring the same kind of information and joyfulness to different users.  A completely free form selection of an axis color doesn't help and can push us in a dead-end, where are no more able to bring back control over the color choices.
2. this doesn't align with our effort of helping the user make the right choices
3. brings us back to the TSVB era, where also the background color was freely selectable by the user and color contrast checks needs to be developed and applied just to that "freedom"
4. this doesn't work out of the box with the dark/light theme mode we currently have: what happens when a user, in light mode, selects `black` as color and the next user visualizes the same chart in dark mode?
 
IMHO you should rediscuss this effort and the possible outcomes